examples/tutorial.ipynb

```diff
@@ -148,7 +148,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Shebang (#!) usage:"
+    "Shebang (#!) and SBATCH options usage:"
```

Comment:
Could we change SBATCH to \`sbatch\`?

Reply:
I wasn't really sure how to phrase that. I don't really know what those comment line options are called. Because "options" could also mean command line options. Script style options?

Reply:
b86e614